src/LandIce/evaluators/LandIce_w_Resid_Def.hpp

```diff
@@ -122,31 +126,28 @@ namespace LandIce
     }
 
 
-    if (d.sideSets->find(sideName) != d.sideSets->end())
+    if (d.sideSetViews->find(sideName)==d.sideSetViews->end()) return;
+
+    auto sideSet = d.sideSetViews->at(sideName);
+    for (int side_idx = 0; side_idx < sideSet.size; ++side_idx)
```

Comment:
Since there is only an implementation for the collapsed layout case, is it safe to assume that the old layouts will never be used for this evaluator? If so, we can probably remove the useCollapsedLayout checks in this evaluator and just use the new layout. 

The useCollapsedLayout stuff is only necessary for evaluators that may be used by multiple problems where one of the problems hasn't been updated yet. I think this one is only used by the coupled problem, right?

Reply:
Good point. The impl is only for collapsed, so attempting to create the ev with non-collapsed layouts might set you up for a nice ride...